src/python/WMCore/JobSplitting/EventAwareLumiByWork.py

```diff
@@ -325,3 +328,29 @@ def populateFilesFromWMBS(self, filesByLocation):
             lumiDict = fileLumis.get(f['id'], {})
             for run in lumiDict.keys():
                 f.addRun(run=Run(run, *lumiDict[run]))
+
+    def getFilesSortedByLocation(self, eventsPerJob):
```

Comment:
Is this function the same as the one in the other splitting code? If so, put it in the base class or import it from one to the other.

Reply:
The EventAware algos have the same code, not the LumiBased. I'm usually not in favor of having these x-imports, so I think I'll just add it to the base class then.